gobblin-core-base/build.gradle

```diff
@@ -21,6 +21,7 @@ apply plugin: 'me.champeau.gradle.jmh'
 
 dependencies {
   compile project(":gobblin-api")
+  compile project(":gobblin-metastore")
```

Comment:
What is this module used for ?

Reply:
FlushingExtractor, which is inside gobblin-core-base, is dependent on StateStoreBasedWatermarkStorage which is being moved to gobblin-metastore as part of this PR.